hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/nativeio/NativeIO.c

```diff
@@ -36,17 +43,91 @@
 #include <sys/resource.h>
 #include <sys/stat.h>
 #include <sys/syscall.h>
-#ifdef HADOOP_PMDK_LIBRARY
```

Comment:
Most of the changes in this file is redundant. We want to keep the existing pmem feature (HDFS-13762).